app/views/users/index.html.erb

```diff
@@ -5,21 +5,21 @@
     <%@users.each do |user|%>
 
     <li class="user">
-      <%=link_to "/users/#{user.id}",class: "user-img", :action => 'go' do%>
-      <div >
-        <img alt="<%= user.name %>'s photo" src=<%=user.photo%>>
-      </div>
-      <% end %>
 
+      <div class="user-img">
+        <%=image_tag("random_face.jpg", class:"user-img", alt: user.name)%>
+      </div>
       <div class='user_item'>
+        <%=link_to "/users/#{user.id}",class: 'user-link', :action => 'go' do%>
         <h2>User <%= user.name %></h2>
+        <% end %>
         <p class='posts-count'>Number of posts : <%= user.posts_counter %></p>
       </div>
 
     </li>
     <%end%>
   </ul>
   <%=link_to "/users/1/posts/new",class: "new-form-btn", :action => 'go' do%>
- <button class="button">New Post</button>
+  <button class="button">New Post</button>
```

Comment:
- [ ] When I try to create a post am getting an error you are missing `layouts/alert` partial
![post](https://user-images.githubusercontent.com/44745855/156752584-50f3e334-f735-4426-9872-56b92da7cd8f.png)

Reply:
- [ ] Remember to check if your flash messages are working on `posts#create`, `comments#create`, and `likes#create`